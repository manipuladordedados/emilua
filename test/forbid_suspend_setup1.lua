this_fiber.allow_suspend()
