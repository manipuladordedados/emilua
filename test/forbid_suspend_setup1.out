Main fiber from VM 0x0 panicked: 'Suspension already allowed'
stack traceback:
	input.lua:1: in main chunk
